src/main/pages/che-7/end-user-guide/proc_converting-a-che-6-workspace-to-a-basic-che-7-devfile.adoc

```diff
@@ -60,20 +59,21 @@ Python with Theia IDE
 | Java Spring Boot
 |===
 
-.. By default the example project is added to your workspace. You can remove it by cliclik a *Remove* button.
+.. By default, the example project is added to the workspace. The default project can be removed by clicking the *Remove* button:
 +
 image::workspaces/remove-default-project.png[link="{imagesdir}/workspaces/remove-default-project.png"]
 
-.. You can import your own project. Click *Add or Import Project*, select Git or Github option and import your project.
+.. One can import a custom project that was used in Che 6 workspace by clicking the *Add or Import Project* and select *Git* or *Github* options:
```

Comment:
I'm not sure, but maybe ` by clicking and selectING` would be better?

Reply:
sure thing, nice catch ;-)